esp-link/cgiwifi.c

```diff
@@ -26,6 +27,27 @@ Cgi/template routines for the /wifi url.
 #define DBG(format, ...) do { } while(0)
 #endif
 
+#define LOG(severity, format, ...)  do {	\
+	char buffer[128];			\
+	os_sprintf(buffer, format, ## __VA_ARGS__);	\
+	syslog(SYSLOG_FAC_USER, severity, "WIFI", buffer);			\
+	DBG("WIFI %s\n", buffer);	\
+} while(0)
```

Comment:
Why? Only the tag is doubled - 4 bytes (aka ptr size). The format string is in buffer[] to avoid duplicate strings.